panoramix/assets/javascripts/modules/panoramix.js

```diff
@@ -217,6 +217,15 @@ var px = (function () {
         others += widget.find('.slice_header').height();
         return widget.height() - others;
       },
+      setResizeOnWindowResize: function () {
+        var resizeTimer;
```

Comment:
nice, then your browser won't crash :) nit I think `bindResizeToWindowResize` is a better name but LGTM :+1: